src/veileder-main.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
```

Comment:
Jeg ville kanskje kalle denne filen for "main-veilder.tsx", slik at den vises under "main.tsx" og gjør det tydelig at vi har to entries.

Reply:
Renamet